pkg/etcd-backup-operator/controller/controller.go

```diff
@@ -0,0 +1,75 @@
+package controller
+
+import (
+	"context"
+	"time"
+
+	api "github.com/coreos/etcd-operator/pkg/apis/etcd/v1beta2"
+
+	"github.com/Sirupsen/logrus"
+	"k8s.io/apimachinery/pkg/fields"
+	"k8s.io/apimachinery/pkg/util/wait"
+	"k8s.io/client-go/tools/cache"
+	"k8s.io/client-go/util/workqueue"
+)
+
+func init(){
+	logrus.WithField("pkg", "controller"),
```

Comment:
Why initialize logrus with such field?
All logrus logging will now contain this field even if it's outside "controller" package.

Reply:
just realized that. removing this in favor of https://github.com/coreos/etcd-operator/blob/master/pkg/controller/controller.go#L76